src/java.base/share/classes/jdk/internal/misc/X-ScopedMemoryAccess.java.template

```diff
@@ -580,11 +580,11 @@ public class ScopedMemoryAccess {
     @ForceInline
     private static
     <V extends VectorSupport.Vector<E>, E, S extends VectorSupport.VectorSpecies<E>>
-    V loadFromMemorySegmentScoped(ScopedMemoryAccess.Scope scope,
-                                  Class<? extends V> vmClass, Class<E> e, int length,
-                                  MemorySegmentProxy msp, long offset,
-                                  S s,
-                                  VectorSupport.LoadOperation<MemorySegmentProxy, V, S> defaultImpl) {
+    V loadFromMemorySegmentScopedInternal(ScopedMemoryAccess.Scope scope,
+                                          Class<? extends V> vmClass, Class<E> e, int length,
+                                          MemorySegmentProxy msp, long offset,
+                                          S s,
+                                          VectorSupport.LoadOperation<MemorySegmentProxy, V, S> defaultImpl) {
```

Comment:
Is it possible to merge this internal method with `loadFromByteBufferScopedInternal` ? And the same to other relative masked and store operations.

Reply:
OK,  that's fine for me. Thanks!